algovivo/System.js

```diff
@@ -72,9 +72,10 @@ class System {
 
     this.setX(data.x);
     
-    const r = ten.zeros([numVertices]);
-    if (this.r != null) this.r.dispose();
-    this.r = r;
+    // const r = ten.zeros([numVertices]);
+    // if (this.r != null) this.r.dispose();
+    // this.r = r;
+    this.r = null;
```

Comment:
Leaving this buffer for now, will reuse it to parameterize masses at some point.